kairos-cli/Cargo.toml

```diff
@@ -13,3 +13,8 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+assert_cmd = "2.0.13"
```

Comment:
this can be added to `[dev-dependencies]` as it is a dependency used in tests only. I'm also not convinced that this dependency is necessary as
one could simply use assert_eq to verify that the output of a command matches, moreover the `cargo_bin("kairos-cli")` can be substituted with `Command::new(env!("CARGO_BIN_EXE_kairos-cli"))`

Reply:
Moved in https://github.com/cspr-rad/kairos/pull/17/commits/fd4476e05a88298df87fb7eb9ca3f14f1bf3c859.